Sources/VLCPlaybackController.h

```diff
@@ -12,7 +12,9 @@
  *****************************************************************************/
 
 #import "VLCEqualizerView.h"
-
+#if TARGET_OS_iOS
+#import <MobileVLCKit/MobileVLCKit.h>
+#endif
 extern NSString *const VLCPlaybackControllerPlaybackDidStart;
```

Comment:
This shouldn't be needed as MVK is imported by the pch.

Reply:
this is actually needed for Swift which doesn't use the pch.